Microsoft.Toolkit.Parsers/Markdown/MarkdownElement.cs

```diff
@@ -10,12 +10,12 @@
 // THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
 // ******************************************************************
 
-namespace Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Parse
+namespace Microsoft.Toolkit.Parsers.Markdown
 {
     /// <summary>
-    /// This is an internal class used as the base class of all markdown elements.
+    /// This is a class used as the base class of all markdown elements.
     /// </summary>
-    internal abstract class MarkdownElement
+    public abstract class MarkdownElement
     {
```

Comment:
Is there another block type we could use here on all elements?  Trying to think if that'd be nicer than having to check class instances.

Also wondering if we want a common 'Text' property or if toString is sufficient.  Or maybe 'DisplayText' which would have the text without the markup?

Reply:
Currently the Text Property belongs to IInlineLeaf, as not all Inlines have text. And Blocks (Which inherit from MarkdownElement definitely don't have a text property.